config/initializers/warden_hooks.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Warden::Manager.after_set_user do |user, auth, opts|
+  scope = opts[:scope]
+  auth.cookies.signed["#{scope}.id"] = user.id
+  auth.cookies.signed["#{scope}.expires_at"] = 30.minutes.from_now
```

Comment:
Question: do we need a user.expires_at cookie? What does it do for us?

Reply:
I'm not sure, that app need expires_at, but in general cases it is needed.

Reply:
_shrug_ it probably won't cause problems, so I'm gonna merge and we'll see how it goes.